cli/src/ui.rs

```diff
@@ -46,6 +46,7 @@ use crate::formatter::LabeledScope;
 use crate::formatter::PlainTextFormatter;
 
 const BUILTIN_PAGER_NAME: &str = ":builtin";
+const ENV_PAGER_NAME: &str = ":envpager";
```

Comment:
nit: just ":env"?

Reply:
I wondered about this, and thought that `:envpager` might be nice since it answers the question of "what part of the environment is relevant here?". Environment could also refer to a dotfile, or something weird.

But my opinion is not very strong, and `:env` is undeniably shorter. We can go with whatever you and Kyle agree on.